docs/arch_develop.md

```diff
@@ -71,39 +71,35 @@ Some components of the core library depend on the `package.json`s of the extensi
     -   Does not restore, it is a superset of what exists in `packages/core` for `configuration.properties`.
     -   To develop for the Amazon Q extension: add all changes to `packages/amazonq/package.json`, EXCEPT for settings that are references by code in the core library, or settings that already exist in the core `package.json`
 
-## Shared vs Common names
+## `web`, `node`, `common`, `shared` naming conventions
```

Comment:
Maybe not in this PR, but "common" should go away, is it needed still?

Reply:
"common" is the overarching name we use to refer to code that works in any evironment. So while we wont be explicitly using it in file/folder names, it will be used in comments and such